config/routes.js

```diff
@@ -5,6 +5,10 @@
  */
 
 module.exports.routes = {
+  // for OIDC
+  // XXX: unclear if its ok to redirect like this, can we somehow directly call the oidc-provider discovery function here?
+  'GET    /.well-known/openid-configuration':             '/noo/oauth/.well-known/openid-configuration',
```

Comment:
...this whole thing is so nicely modular that it'd be cool if this route could somehow get snuck-into the oidc-provider routing above, but I bet you thought of that and likely it was more trouble than it's worth--if even possible.